bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-message-form/component.tsx

```diff
@@ -40,9 +41,11 @@ import { GraphqlDataHookSubscriptionResponse } from '/imports/ui/Types/hook';
 import { throttle } from '/imports/utils/throttle';
 import logger from '/imports/startup/client/logger';
 import { CHAT_EDIT_MESSAGE_MUTATION } from '../chat-message-list/page/chat-message/mutations';
+import ChatMentionPicker, { ChatMentionPickerRef } from './chat-mention-picker/component';
 
 const CLOSED_CHAT_LIST_KEY = 'closedChatList';
 const START_TYPING_THROTTLE_INTERVAL = 1000;
+const MENTION_REGEX = /@([^\s@]*)/gi;
```

Comment:
```suggestion
const MENTION_REGEX = /\[@[^\]]+\]\(bbb:\/\/\w+\)/gi;
```

My suggestion (I didn't test).

Reply:
You are right, I thought this REGEX was used for other purpose, sorry!